flo_telepresence/src/display_cv.py

```diff
@@ -49,9 +52,9 @@ def draw_text(img,  # pylint: disable=too-many-arguments
     if text == '':
         return 0
 
-    x_pos, y_pos = pos  # pylint: disable=invalid-name
-    x_pos = int(math.ceil(x_pos))
-    y_pos = int(math.ceil(y_pos))
+    x, y = pos  # pylint: disable=invalid-name
```

Comment:
This has a pylint exception, I don't understand why it's still throwing up a linting error

Reply:
I think if you put it on a prior line, then it will be ok:
```{python}
    # pylint: disable=invalid-name
    x, y = pos
```